src/mudlet.cpp

```diff
@@ -5363,3 +5363,17 @@ void mudlet::sanitizeUtf8Path(QString& originalLocation, const QString& fileName
     }
 }
 #endif
+
+// Enable redirects and HTTPS support for a given url
+void mudlet::setNetworkRequestDefaults(const QUrl& url, QNetworkRequest& request)
+{
+    request.setAttribute(QNetworkRequest::RedirectPolicyAttribute, QNetworkRequest::NoLessSafeRedirectPolicy);
+
+    request.setRawHeader(QByteArray("User-Agent"), QByteArray(QStringLiteral("Mozilla/5.0 (Mudlet/%1%2)").arg(APP_VERSION, APP_BUILD).toUtf8().constData()));
+#ifndef QT_NO_SSL
```

Comment:
:thinking: Actually - instead of this compilation option - should we not instead move to using `(bool) QSslSocket::supportsSsl()` to check for it at run-time. Or can that test only be used if `QT_NO_SSL` is not defined at compile time anyhow?